index.css

```diff
@@ -0,0 +1,110 @@
+{box-sizing: border-box}
+
+
+
+body{
+
+}
+
+nav{
```

Comment:
need a space between nav and the curly braces (please address this throughout your whole css file)